### PR TITLE
when using admin permission, explicitly name it

### PR DESCRIPTION
**Pull request by maxtaco**
- this is needed to work with keybase/keybsae CORE-5469 on the server-side

### CODE REVIEW

File: go/teams/member_set.go [hunk 1]
```diff
@@ -1,10 +1,9 @@
 package teams
 
 import (
-	"golang.org/x/net/context"
-
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/keybase1"
+	"golang.org/x/net/context"
```
Comment (patrickxb) [inline go/teams/member_set.go]: our goimports are going to fight over this...mine will move it back to where it was.

Comment (maxtaco) [inline go/teams/member_set.go]: oh crap

File: go/teams/teams.go [hunk 2]
```diff
@@ -268,7 +268,10 @@ func (t *Team) Rotate(ctx context.Context) error {
 	memSet := newMemberSet()
 
 	// create the team section of the signature
-	section, err := memSet.Section(t.Chain.GetID())
+
+	// TODO -- fill in a non-nil if we can't write/admin this chain explicitly and need
+	// admin permissions. See CORE-5501
+	section, err := memSet.Section(t.Chain.GetID(), nil)
```
Comment (patrickxb) [inline go/teams/teams.go]: why doesn't this try calling t.getAdminPermission()?

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits 8cf511862070...b0189bedeea1 on linux. <details><summary>:mag_right: 6 errored, 1 changed</summary>

 * Error in 8cf511862070: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in b0189bedeea1: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 8cf511862070: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in b0189bedeea1: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 8cf511862070: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Error in b0189bedeea1: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Changed: **Markdown-emoji** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497625183657-8cf511862070-b0189bedeea1-linux/Markdown-emoji-8cf511862070.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497625183657-8cf511862070-b0189bedeea1-linux/Markdown-emoji-b0189bedeea1.png)  
   Markdown-emoji comparison
</summary>

File: go/teams/teams.go (hunk 2 above)
Comment (maxtaco) [inline go/teams/teams.go]: ok will do

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits 8cf511862070...c43d93ac71a3 on linux. <details><summary>:mag_right: 6 errored, 1 changed</summary>

 * Error in 8cf511862070: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in c43d93ac71a3: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 8cf511862070: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in c43d93ac71a3: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 8cf511862070: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Error in c43d93ac71a3: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497627384789-8cf511862070-c43d93ac71a3-linux/Tracker-New_user_Scroll_1-8cf511862070.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497627384789-8cf511862070-c43d93ac71a3-linux/Tracker-New_user_Scroll_1-c43d93ac71a3.png)  
   Tracker-New_user_Scroll_1 comparison
</summary>

**Comment by maxtaco** [comment]
ok @patrickxb made changes

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits 8cf511862070...c43d93ac71a3 on linux. <details><summary>:mag_right: 6 errored</summary>

 * Error in 8cf511862070: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in c43d93ac71a3: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 8cf511862070: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in c43d93ac71a3: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 8cf511862070: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Error in c43d93ac71a3: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
</summary>

**Pull request merged**